docfx_yaml/extension.py

```diff
@@ -1002,6 +1031,43 @@ def pretty_package_name(package_group):
     return " ".join(capitalized_name)
 
 
+# Given generated markdown files, incorporate them into the docfx_yaml output.
+# The markdown file metadata will be added to top level of the TOC.
+def find_markdown_pages(app, outdir):
+    # Use this to ignore markdown files that are unnecessary.
+    files_to_ignore = [
+        "index.md",     # merge index.md and README.md and index.yaml later
+        "reference.md", # Reference docs overlap with Overview. Will try and incorporate this in later.
+        "readme.md",    # README does not seem to work in cloud site
+        "upgrading.md", # Currently the formatting breaks, will need to come back to it.
```

Comment:
File issue and reference it here? Possibly the same issue as `reference.md`?

Reply:
Done. Different issue, `upgrading.md` file already comes as markdown format in `docs/` directory, whereas other files come in reStructuredText format (`.rst`). Processing `.rst`->`.md` with doc-pipeline seems to work well, but processing `.md` -> `.md`(?) -> doc-pipeline seems to break. Will need to mitigate this in the middle and use markdown from the source directly, or to make this work on doc-pipeline.